ares/n64/aleck64/aleck64.hpp

```diff
@@ -0,0 +1,134 @@
+struct Aleck64 : Memory::RCP<Aleck64> {
+  Node::Object dipSwitchNode;
+
+  struct Writable : public Memory::Writable {
+    Aleck64& self;
+
+    Writable(Aleck64& self) : self(self) {}
+
+    template<u32 Size>
+    auto writeBurst(u32 address, u32 *value, const char *peripheral) -> void {
+      if (address >= size) return;
```

Comment:
This code is copied from rdram, but rdramlives at n64 address `0x0` so this logic works, you have to mask off the address here or else your reads & writes and address logic will fly off to space. Super Real Mahjong seems to read/write to 0xc000'0000 right at boot causing the crash. Probably will fix other games too, just later on in the gameplay.

```suggestion
    auto writeBurst(u32 address, u32 *value, const char *peripheral) -> void {
      address = address & 0x00ff'ffff;
      if (address >= size) return;
```

Reply:
Most games don't use this ram and just allocate it which is likely why it wasn't noticed yet, heh, thanks!